creator/studies/models.py

```diff
@@ -192,6 +222,14 @@ class Meta:
     def __str__(self):
         return self.kf_id
 
+    def save(self, *args, **kwargs):
```

Comment:
I don't think this should be here bc this will prevent us from ever being able to set the program explicitly to None (like in the undo function of the migration). This type of logic should probably go in the study create mutation